projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/query-management/models/qualifying-questions/casetype-qualifying-questions.model.ts

```diff
@@ -0,0 +1,6 @@
+import { QualifyingQuestion } from './qualifying-question.model';
+
+export interface CaseTypeQualifyingQuestions {
```

Comment:
casetype id should be the key for an object. Thus:

```
export type CaseTypeQualifyingQuestions = { [caseTypeId: string]: QualifyingQuestion[] };

Reply:
Note that your interface would be sub-optimal, as it requires looping through the whole set and trying to match by name. By making it an object where the id is the key, you can immediately refer to it.

Reply:
@ronaldmansveld I have now updated the code as you suggested.